src/config/messages.js

```diff
@@ -14,6 +14,7 @@ const ERROR_SETTING_DATABASE = 'There was an error setting the database';
 const ERROR_GETTING_DATABASE = 'There was an error retrieving the database';
 const ERROR_DELETING_DATASET_MESSAGE =
   'There was an error deleting the dataset';
+const ERROR_ADDING_DATASET_MESSAGE = 'There was an error adding the dataset';
```

Comment:
Go with `ERROR_LOADING_DATASET_MESSAGE = 'There was an error loading the dataset';`